airflow/models/variable.py

```diff
@@ -47,12 +47,14 @@ class Variable(Base, LoggingMixin):
     id = Column(Integer, primary_key=True)
     key = Column(String(ID_LEN), unique=True)
     _val = Column('val', Text)
+    description = Column(String(5000))
```

Comment:
```suggestion
    description = Column(Text(5000))
```

This will give you `textarea` instead of `input`

Reply:
Perfect! Thanks a lot!